tests/foreman/cli/test_host.py

```diff
@@ -2088,9 +2088,9 @@ def setUpClass(cls):
         # create a rh capsule content
         setup_org_for_a_rh_repo(
             {
-                'product': PRDS['rhsc'],
-                'repository-set': REPOSET['rhsc7'],
-                'repository': REPOS['rhsc7']['name'],
+                'product': PRDS['rhel'],
+                'repository-set': REPOSET['rhst7_66'],
```

Comment:
@tstrych should we use `Satellite Tools 6.7` in our tests instead of 6.6?

Reply:
@jameerpathan111 , not sure if satellite tools 6.7 are published, if it will work, but yes I can add them also to constants :)

Reply:
@tstrych that's wrong. Even if they are not published, we're supposed to use the tested one - this would mean, using the one from the dogfood (defined as custom repository)

Reply:
Hello

I think @rplevka means that in the time before the tools repo is published for the next release, which for some reason always happens late in the development cycle, tests related to packages in the tools repo should use the tools repo from dogfooding server.

I took a look to see where we pull from dogfood. I found two places which use 'settings.sattools_repo' from robottelo.properties file:

https://github.com/SatelliteQE/robottelo/blob/8907e0257eb0dd978624db7f87e647f026c76aa3/robottelo/cli/factory.py#L3419

https://github.com/SatelliteQE/robottelo/blob/8907e0257eb0dd978624db7f87e647f026c76aa3/robottelo/products.py#L647